config.ini.TEMPLATE

```diff
@@ -0,0 +1,6 @@
+[DISCORD]
+token=<<DISCORD TOKEN HERE>>
+invite-url=<<INVITE URL HERE>>
```

Comment:
Is this file mentioned in the README.md that it needs setting up in the run/install instructions.

Reply:
> Joel are you still managing the deploying to prod? Perhaps on the weekend, you can demonstrate the deployment process to the new devs at the same time.

Not anymore no. It's now run by FBaPS staff so when we want to push a new change to prod we need to send an email to Alun Jones auj@aber.ac.uk

Reply:
@Amheus can you make sure that the readme is up to date so that deployment from Alun's side is easy and straight forward?

Reply:
aye will do